charts/bpa/values.schema.json

```diff
@@ -470,7 +470,15 @@
                             "type": "boolean"
                         }
                     }
+                },
+                "ux": {
+                    "type": "string",
+                    "enum": ["bosch", "bcgov"],
+                    "description": "The UX look and feel configuration",
```

Comment:
Currently I used `description` and `title` only if I need it in the form.
Reason: schema is not seen by other users , the look more into the values file and the Readme.md which is generated out of the values file. 
Unfortunately currently in helm there is values file and schema file.

Reply:
removed.